lms/djangoapps/courseware/tests/test_inline_analytics_utils.py

```diff
@@ -78,3 +78,41 @@ def test_multi_responses(self):
         self.assertEquals(response[1].response_type, 'radio')
         self.assertEquals(response[0].message, None)
         self.assertEquals(response[1].message, None)
+        self.assertEquals(response[0].choice_name_list, '[]')
+        self.assertEquals(response[1].choice_name_list, '[&quot;choice_0&quot;, &quot;choice_1&quot;, &quot;choice_2&quot;]')
+
+    def test_multi_responses_name_mask(self):
+
+        xml = textwrap.dedent("""\
```

Comment:
Just an FYI: I don't think you need the `\` continuation for code inside of parens, particularly with docstrings.